packages/@aws-cdk/aws-s3/lib/auto-delete-objects-handler/index.ts

```diff
@@ -0,0 +1,43 @@
+export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent, s3?: any) {
```

Comment:
Just out of curiosity, why are we not hoisting the `s3` object from `aws-sdk` on top of handler instead of accepting it as argument? Is this required to facilitate unit testing?